tests/unit/spire_test.go

```diff
@@ -0,0 +1,48 @@
+package unit_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	helmchart "helm.sh/helm/v3/pkg/chart"
+	helmloader "helm.sh/helm/v3/pkg/chart/loader"
+	helmutil "helm.sh/helm/v3/pkg/chartutil"
+	helmengine "helm.sh/helm/v3/pkg/engine"
+)
+
+func ValueStringRender(chart *helmchart.Chart, values string) (map[string]string, error) {
+	v, err := helmutil.ReadValues([]byte(values))
+	if err != nil {
+		return nil, err
+	}
+	ro := helmutil.ReleaseOptions{Name: "spire", Namespace: "spire-server", Revision: 1, IsUpgrade: false, IsInstall: true}
+	v, err = helmutil.ToRenderValues(chart, v, ro, helmutil.DefaultCapabilities);
+	if err != nil {
+		return nil, err
+	}
+	objs, err := helmengine.Render(chart, v)
+	return objs, err
+}
+
+var _ = Describe("Spire", func() {
+	chart, err := helmloader.Load("../../charts/spire")
+	if err != nil {
+		panic(err)
```

Comment:
Is panic on error normal for this unit test framework? Or is there an `Expect(err).ToBeNil` or something like that?

Reply:
Ah yes for other framework I normally use there is a `Expect.NoError(err)`. Probably nicer and might do some nice logging as well.

Reply:
./spire_test.go:29:9: Expect.NoError undefined (type func(actual interface{}, extra ...interface{}) "github.com/onsi/gomega/types".Assertion has no field or method NoError)...

Reply:
Docs make it sound like this should do the trick:
        Expect(err).Should(Succeed())

Reply:
How about `Expect(err).ShouldNot(HaveOccurred())`